you-have-mail-android/app/src/main/java/dev/lbeernaert/youhavemail/MainActivity.kt

```diff
@@ -308,8 +314,42 @@ fun ShowSettingDialog(context: Context, openDialog: MutableState<Boolean>) {
                         Text(stringResource(id = R.string.ok))
                     }
                 }
-
             },
         )
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun ShowErrorDialog() {
+    AlertDialog(
+        onDismissRequest = {},
+        title = {
+            Text(
+                modifier = Modifier.fillMaxWidth(),
+                textAlign = TextAlign.Center,
+                fontSize = 20.sp,
+                text = "Error"
+            )
+        },
+        text = {
+            Column() {
```

Comment:
Could you also center the message text here?

Reply:
ok, did that with textAlign = TextAlign.Center,